spec/license_wrap_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'word wrapping' do
+  Dir["#{licenses_path}/*.txt"].each do |file|
```

Comment:
You *may* be able to do this with `licenses.each do |license|` and then `license.name` and `license.body` below since we [read in the licenses in the helper](https://github.com/github/choosealicense.com/blob/gh-pages/spec/spec_helper.rb#L36-L43). We had to manually read in the files via the BOM test to avoid errors.

Reply:
Oh right. `license['slug']` and `license['content']` it seems. Thanks!